libgnucash/backend/xml/io-gncxml-v1.cpp

```diff
@@ -657,7 +657,7 @@ glist_kvp_value_end_handler (gpointer data_for_children,
         sixtp_child_result* cr = (sixtp_child_result*) lp->data;
         KvpValue* kvp = (KvpValue*) cr->data;
 
-        /* children are in reverse chron order, so this fixes it. */
+        /* children are in reverse cron order, so this fixes it. */
```

Comment:
No, chron here is shorthand for chronological, not referring to the cron tool.

Reply:
would you object to `chronological`?

Reply:
No, that's fine.

Reply:
https://github.com/Gnucash/gnucash/pull/634/files#diff-d2acbd5839cd84ca0ea14906500fa669R660